src/DssSpell.t.sol

```diff
@@ -2476,4 +2276,23 @@ contract DssSpellTest is DSTest, DSMath {
         }
         assertEq(expectedHash, actualHash);
     }
+
+    function test_RWA002_values() public {
+        vote(address(spell));
+        spell.schedule();
+        hevm.warp(spell.nextCastTime());
+        spell.cast();
+        assertTrue(spell.done());
+
+        bytes32 ilk = bytes32("RWA002-A");
+
+        // Confirm pip value.
+        DSValueAbstract pip = DSValueAbstract(addr.addr("PIP_RWA002"));
+        assertEq(uint256(pip.read()), 22_495_725 * WAD);
```

Comment:
We don't need to do it right now, but I'll add a card to add a generalized pip test for RWA above.

Reply:
might be too much but can we test here that debt can be drawn up to the ceiling (and not more)?

Reply:
ah this might be what @brianmcmichael meant (missed his comment), can do it later.

Reply:
Hmmmm we could by giving ourselves authority over the RWA002 urn. Not sure if we strictly need it here though.